dspace/config/modules/usage-statistics.cfg

```diff
@@ -40,4 +40,21 @@ usage-statistics.authorization.admin.workflow=true
 
 # Enable/disable if a matching for a bot should be case sensitive
 # Setting this value to true will increase cpu usage, but bots will be found more accurately
-#usage-statistics.bots.case-insensitive = false
\ No newline at end of file
+#usage-statistics.bots.case-insensitive = false
+
+# Configuration parameters for anonymizing statistics
+
+# Anonymize statistics the moment they are created
+# anonymize_statistics.anonymize_on_log = false
```

Comment:
The space before the config settings with the default values should be removed, so that there is a difference between comments and default config values.